### PR TITLE
Sync dev-es branch with main

### PR DESCRIPTION
**Pull request by krol3**
### Describe your changes

Merge main into dev-es to keep in sync

### CODE REVIEW

File: i18n/es.toml [hunk 1]
```diff
@@ -18,9 +18,14 @@ other = "Buscar en este sitio…"
 [ui_in]
 other = "en"
 
-# Used in sentences such as "All Tags"
-[ui_all]
-other = "todo(a)"
+# Phrases for tags
+[ui_see_all_tags]
+other = "See all tags"
```
Comment (electrocucaracha) [inline i18n/es.toml:23]: Eventually we have to translate this entry

**Review by electrocucaracha — APPROVED** [review]

File: i18n/es.toml (hunk 1 above)
Comment (krol3) [reply i18n/es.toml:23]: @electrocucaracha  can you create an issue to track this?

**Review by seokho-son — APPROVED** [review]
LGTM !

File: i18n/es.toml (hunk 1 above)
Comment (electrocucaracha) [reply i18n/es.toml:23]: Done in #1748

**Pull request merged**